common.sh

```diff
@@ -55,6 +55,7 @@ if [ -z "${OPENSHIFT_RELEASE_IMAGE}" ]; then
   LATEST_CI_IMAGE=$(curl https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/4.4.0-0.ci/latest | grep -o 'registry.svc.ci.openshift.org[^"]\+')
 fi
 export OPENSHIFT_RELEASE_IMAGE="${OPENSHIFT_RELEASE_IMAGE:-$LATEST_CI_IMAGE}"
+export OPENSHIFT_VERSION=$(echo $OPENSHIFT_RELEASE_IMAGE | sed "s/.*release:\([[:digit:]].[[:digit:]]\).*/\1/")
```

Comment:
assuming `release:` might bite us at some point, but I don't have an example of where it would break at the moment.  Maybe something like this so it has a fallback?

`export OPENSHIFT_VERSION=${OPENSHIFT_VERSION:-$(echo $OPENSHIFT_RELEASE_IMAGE | sed "s/.*release:\([[:digit:]].[[:digit:]]\).*/\1/")}`

Reply:
Hmm good point, perhaps just split on the `:` e.g `.*:[[:digit:]].[[:digit:]]` ?

I could test to ensure the resulting OPENSHIFT_VERSION is non-empty and default to 4.4 if for some reason it is too, if we think that's needed.

Reply:
@russellb I relaxed the regex to just split on the `:` - let me know if you think that's enough or if we need to also have a check that defaults to 4.4 (I was hoping to avoid this since it'll still potentially download the wrong version if the regex fails)

Reply:
works for me